Gemfile.lock

```diff
@@ -256,11 +256,11 @@ GEM
     crass (1.0.6)
     css_parser (1.7.1)
       addressable
-    daemons (1.3.1)
+    daemons (1.4.0)
```

Comment:
are we using this anymore now that we run background jobs via `Procfile` in dev?

Reply:
should be safe, yep. Deployed environments use `rake jobs:work` which doesn't need it either. Removed in 11d3a7a